client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -61,18 +61,20 @@ export default class ProductFormDetailsCard extends Component {
 						/>
 					</FormLabel>
 				</div>
-				<div>
+				<div className="products__product-form-details-wrapper">
+					<div className="products__product-form-details-images">
 
-				</div>
-				<div>
-					<FormLabel>
-						<span>{ __( 'Product name' ) }</span>
-						<FormTextInput value={ product.name || '' } onChange={ this.setName } />
-					</FormLabel>
-					<FormLabel>
-						<span>{ __( 'Description' ) }</span>
```

Comment:
Agreed. Fixed in https://github.com/Automattic/wp-calypso/pull/13251/commits/9f6e9e2f91910bf0482a7032f163727fe16a7a79